src/influxdb_ioxd.rs

```diff
@@ -59,11 +49,21 @@ pub enum Error {
     #[snafu(display("Error serving RPC: {}", source))]
     ServingRPC { source: self::rpc::Error },
 
-    #[snafu(display("Specifed {} for the object store, but not a bucket", object_store))]
-    InvalidCloudObjectStoreConfiguration { object_store: ObjStoreOpt },
+    #[snafu(display(
+        "Specifed {} for the object store, required configuration missing for {}",
```

Comment:
```suggestion
        "Specified {} for the object store, required configuration missing for {}",
```

 I realize you did not introduce this typo, but I noticed it while playing around